jupyterhub/handlers/pages.py

```diff
@@ -67,6 +67,12 @@ def get(self):
         if user.running:
             # trigger poll_and_notify event in case of a server that died
             yield user.spawner.poll_and_notify()
+
+        # send user to /hub/user/:name if spawner is pending
+        if user.spawner.pending:
+            url = url_path_join(self.hub.base_url, user.url)
+            self.redirect(url)
```

Comment:
This should not be a redirect, but instead setting the url variable used immediately below. We shouldn't ever automatically redirect away from `/hub/home`, because that's how you get to the stop button, admin page, etc.